admin/salemaker.php

```diff
@@ -259,10 +238,10 @@ function SetCategories() {
     </head>
 <?php } ?>
 <?php if ($action == 'new' || $action == 'edit') { ?>
-       <body onload="SetCategories(); SetFocus(); init()">
+       <body onload="SetCategories();">
        <div id="spiffycalendar" class="text"></div>
 <?php } else { ?>
-       <body onload="SetFocus(); init()">
+       <body>
```

Comment:
Removing `SetFocus()` here means line 307 should have `autofocus` added to the input field attributes, shouldn't it?

Reply:
I'll check